lib/ansible/modules/packaging/os/yum.py

```diff
@@ -1046,7 +1046,7 @@ def ensure(module, state, pkgs, conf_file, enablerepo, disablerepo,
     if state in ['installed', 'present', 'latest']:
 
         if module.params.get('update_cache'):
-            module.run_command(yum_basecmd + ['makecache'])
+            module.run_command(yum_basecmd + ['clean expire-cache'])
```

Comment:
This should have the two arguments in separate strings in case the implementation that invokes the commands changes to exec commands in the future.